data/tlg0006/tlg009/tlg0006.tlg009.perseus-grc2.xml

```diff
@@ -2195,9 +2195,9 @@
                             <l n="1170">σῴσας με νέρθεν, ἦλθον, εἴ τι δεῖ, γέρον,</l>
                             <l n="1171">ἢ χειρὸς ὑμᾶς τῆς ἐμῆς ἢ συμμάχων.</l>
                             <l n="1172">ἔα·</l>
-                            <l n="1173">τί νεκρῶν τῶνδε πληθύει πέδον;</l>
-                            <l n="1174">οὔ που λέλειμμαι καὶ νεωτέρων κακῶν</l>
-                            <l n="1175">ὕστερος ἀφῖγμαι; τίς τάδʼ ἔκτεινεν τέκνα;</l>
+                            <l n="1172a">τί νεκρῶν τῶνδε πληθύει πέδον;</l>
```

Comment:
Hi, @balmas In this instance, line 1172 starts with ἔα and ends with πέδον.
It's just that there is an indentation in the page formatting.
I haven't worked on much drama, so I don't know the best solution to this offhand.